Dockerfile

```diff
@@ -22,6 +25,16 @@ RUN --mount=type=cache,target=/root/.cargo/git/db \
     cp target/$(xx-cargo --print-target-triple)/release/typst target/release/typst && \
     xx-verify target/release/typst
 
-FROM alpine:latest
-WORKDIR /root/
+FROM alpine:3.19
+LABEL org.opencontainers.image.authors="Typst <hello@typst.app>"
```

Comment:
This should be the same name as in the `Cargo.toml`, you can, however, keep the email address